interface.go

```diff
@@ -309,6 +348,29 @@ func (f *Interface) reloadFirewall(c *config.C) {
 		Info("New firewall has been installed")
 }
 
+func (f *Interface) reloadSendRecvError(c *config.C) {
+	stringValue := c.GetString("listen.send_recv_error", "")
+	if (c.InitialLoad() && stringValue != "") || c.HasChanged("listen.send_recv_error") {
+		switch stringValue {
+		case "always":
+			f.sendRecvErrorConfig = sendRecvErrorAlways
```

Comment:
If I'm reading this correctly, the switch statement will only execute in the case of InitialLoad if the value is set to something. If the value is not set...go will implicitly set f.sendRecvErrorConfig to an initial value of 0, which happens to match sendRecvErrorAlways.

I think it'd be nice to explicitly set the default value, though. Maybe edit line 352 to say `stringValue := c.GetString("listen.send_recv_error", "always")` ?

Reply:
good point, I think I have cleaned this up with the latest commit if you can review again.